requirements-3.5.txt

```diff
@@ -0,0 +1,36 @@
+# Required for fairlearn
+matplotlib>=3.0.3
```

Comment:
I wonder if there's a way of making sure that the `requirements` files remain usefully diffable

Reply:
What do you mean?

Reply:
Basically, we want to make sure that any differences between the requirements files are intentional due to issues like this. And not accidental due to us forgetting to update one of the files.

Ideally this would be done with a `diff requirements.txt requirements-3.5.txt` command (adding PowerShell bits if required). However, unless we're also very careful about how we add new dependencies to the files, then we could trip ourselves up by listing new dependencies in different orders.

I'm worrying in advance here; for now we might just take the note to 'be careful.'

Reply:
Yes it works for now...
```
diff (Get-Content .\requirements.txt) (Get-Content .\requirements-3.5.txt)

InputObject       SideIndicator
-----------       -------------
matplotlib>=3.0.3 =>
scipy>=1.3.1      =>
matplotlib>=3.1.3 <=
scipy>=1.4.1      <=
```
I am somewhat hopeful that we're not adding too many new dependencies so it shouldn't ever get to that stage.